### ISSUE TITLE
"panel.addClass is not a function" error in admin product attribute edit page

### ISSUE THREAD

**Issue by textarea** [magento/magento2#23117]
### Preconditions (*)
1. Magento version 2.3.1

### Steps to reproduce (*)
1. Go to menù item STORES -> Attributes -> Product
2. Click on any attribute

### Expected result (*)
1. No javascript errors in browser console

### Actual result (*)
1. I get this error in browser console:
Uncaught TypeError: panel.addClass is not a function
    at checkOptionsPanelVisibility ((index):865)
    at bindAttributeInputType ((index):876)
    at HTMLDocument.<anonymous> ((index):1129)
    at fire (jquery.js:3232)
    at Object.add [as done] (jquery.js:3291)
    at jQuery.fn.init.jQuery.fn.ready (jquery.js:3542)
    at jQuery.fn.init (jquery.js:2967)
    at new jQuery.fn.init (jquery-migrate.js:241)
    at jQuery (jquery.js:75)
    at (index):1128

**Comment by AlexWorking** [comment]
Hello, @textarea. We've been not able to reproduce Your issue. Would You please try to do it Yourself on a clear Magento instance as You are advised to do above:
![Advised](https://user-images.githubusercontent.com/27814011/58871114-19731900-86ca-11e9-9a25-5248085d923e.png)
Thank You.

**Comment by textarea** [comment]
@magento give me 2.3-develop instance - upcoming 2.3.x release

**Comment by magento-engcom-team** [comment]
Hi @textarea. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @textarea, here is your [Magento instance](https://i-23117-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23117-2-3-develop.instances.magento-community.engineering/admin](https://i-23117-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by textarea** [comment]
Hi @AlexWorking, I'm not able to reproduce it too.
I'm closing this issue.
Thanks

**Comment by textarea** [comment]
@magento give me 2.3-develop instance - upcoming 2.3.x release

**Comment by magento-engcom-team** [comment]
Hi @textarea. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @textarea, here is your [Magento instance](https://i-23117-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23117-2-3-develop.instances.magento-community.engineering/admin](https://i-23117-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by textarea** [comment]
@AlexWorking I've found the case the issue can reproduce, simply disable swatches module:
bin/magento module:disable Magento_SwatchesGraphQl Magento_Swatches

then you will able to reproduce the issue

**Comment by AlexWorking** [comment]
@textarea, here is gathered what I've done in one picture:
![cannot](https://user-images.githubusercontent.com/27814011/58954862-06804780-87a3-11e9-8f3b-a55307b27c14.png)
The issue is still not reproduced as You can see. Have I missed something?

**Comment by textarea** [comment]
@AlexWorking I'm in developer mode but nothing else. Please make sure your cache is cleared.
Why I can't see "Storefront properties" tab in your screenshot?
I think your configuration is not loading vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/attribute/js.phtml file.
![Schermata-2019-06-05-alle-14 18 54](https://user-images.githubusercontent.com/1927403/58955837-3593ba80-879d-11e9-83fc-f9cbc2602eb6.jpg)

**Comment by AlexWorking** [comment]
@textarea I repeated everything from the very beginning. So I installed a vanilla Magento instance, version 2.3-develop. Then run these bin/magento commands in this order:
deploy:mode:set developer,
module:disable Magento_SwatchesGraphQl Magento_Swatches --clear-static-content,
indexer:reindex,
cache:flush.
Then opened other attribute that had "Storefront Properties" and the result was the same:
![catid](https://user-images.githubusercontent.com/27814011/58961516-f9b72000-87b1-11e9-8df3-3eeaa3bfdccf.png)
As for the "Storefront Properties" absence in the attribute I used on the previous screenshot it took place because of the very attribute - swatch_image. Looks like this attribute do not have those properties at all.
